bpfman/src/types.rs

```diff
@@ -449,6 +572,16 @@ impl ProgramData {
         })
     }
 
+    /// Retrieves the location of the program.
```

Comment:
```suggestion
    /// Retrieves the location of the program. The Location of the program describes if the program's bytecode was provided via a fully qualified local path OR via an OCI compliant container image tag.
```

Or something like that

Reply:
One thing I noted when viewing the converted documentation was that the first line shouldn't be too long. How about?

/// Retrieves the location of the program.
/// 
/// The location describes whether the program's bytecode was provided 
/// via a fully qualified local path or an OCI-compliant container image tag.

Reply:
Updated in db1fb0d3a133b0da7411f409e2ab5d434b4b3805.